### DISCUSSION TITLE
How to change the coordinate system?

### DISCUSSION THREAD

**Discussion by HBG8267** [pixijs/pixijs#9688] [Q&A]
I just learned to use PIXI.js. I find the default coordinate system is based on the upper-left corner，but now I want it based on the bottom left corner，What should I do？
![image](https://github.com/pixijs/pixijs/assets/16015521/6e29ff3f-8095-4e72-8946-06ba68489c21)

**Comment by SuperSodaSea** [comment]
Try this:

```js
app.stage.scale.set(1, -1);
app.stage.y = app.screen.height;
```

**Reply by HBG8267** [reply]
Thank you for your reply, but the graphics are not symmetrical. After rotation, they are not what I want.
